File: Data Structures & Algorithms/rotate-matrix/submission-0.java
class Solution {
    public void rotate(int[][] matrix) {

        int n = matrix.length ; 
        if( n <= 1) return ; 

        int[][] matrice = new int[n][n];

        for(int i = 0 ; i < n ; i++)
        {
            for(int j = 0 ; j < n ; j++)
            {   
              matrice[i][n-1-j] = matrix[j][i];
                
            }
        }
         for(int i = 0 ; i < n ; i++)
        {
            for(int j = 0 ; j < n ; j++)
            {   
              matrix[i][j] = matrice[i][j];
                
            }
        }
        
    }
}

File: Data Structures & Algorithms/set-zeroes-in-matrix/submission-0.py
class Solution:
    def setZeroes(self, matrix: List[List[int]]) -> None:

        theZeroes :List[int] = [] 


        line : int = len(matrix) ; column : int = len(matrix[0])


        for i in range(line):
            for j in range(column):
                if matrix[i][j] == 0 : theZeroes.append([i,j])


        for arr in theZeroes : 
            matrix[arr[0]] = [0]*column

            for i in range(line):
                matrix[i][arr[1]] = 0 


        return None                


        